### PR TITLE
Add switch expression example

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes https://github.com/dotnet/docs/issues/21140

### CODE REVIEW

File: spec/enums.md [hunk 1]
```diff
@@ -197,3 +197,41 @@ Enum members have the type of their containing enum type (except within other en
 The following operators can be used on values of enum types: `==`, `!=`, `<`, `>`, `<=`, `>=` ([Enumeration comparison operators](expressions.md#enumeration-comparison-operators)), binary `+` ([Addition operator](expressions.md#addition-operator)), binary `-` ([Subtraction operator](expressions.md#subtraction-operator)), `^`, `&`, `|` ([Enumeration logical operators](expressions.md#enumeration-logical-operators)), `~` ([Bitwise complement operator](expressions.md#bitwise-complement-operator)), `++` and `--` ([Postfix increment and decrement operators](expressions.md#postfix-increment-and-decrement-operators) and [Prefix increment and decrement operators](expressions.md#prefix-increment-and-decrement-operators)).
 
 Every enum type automatically derives from the class `System.Enum` (which, in turn, derives from `System.ValueType` and `object`). Thus, inherited methods and properties of this class can be used on values of an enum type.
+
+Enums can be used in [switch expressions](/dotnet/csharp/language-reference/operators/switch-expression). The following example translates values from an enum representing visual directions in an online map to the corresponding cardinal direction:
```
Comment (Youssef1313) [inline spec/enums.md:201]: - The link won't work from GitHub.
- Isn't this the C# 6.0 specs? If so, it shouldn't mention a feature from later versions.

**Comment by 333fred** [comment]
The language spec shouldn't be modified here. I believe @BillWagner has a doc or set of docs that would be more appropriate for this.

**Comment by tdykstra** [comment]
@BillWagner The [issue](https://github.com/dotnet/docs/issues/21140) referred to this page -- should we just close the issue as won't fix, then?

**Comment by BillWagner** [comment]
@tdykstra I'd like to keep that issue open. Soon, we'll have the standard in a public repo, and I'll move all the spec related issues there. Then, the committee will decide what action to take.

**Pull request closed without merging**